docs/css/src/sub-nav.css

```diff
@@ -0,0 +1,26 @@
+#sub-nav {
+  padding-left: calc(var(--unit-space) * 2);
+  border-left: calc(var(--unit-space) / 2) solid var(--color-default);
+  color: var(--color-hint);
+  font-size: .875em;
+}
+
+#sub-nav #nested-sub-nav {
```

Comment:
ID is ok because ola want it like that, but two IDs? or want ola also that?

Reply:
no... just the root of the module should have an id to indicate a module. sorry, if there was a misunderstanding.